src/features/movies/MovieDetails/index.js

```diff
@@ -15,71 +18,84 @@ import {
   TagsWrapper,
   AdditionalInfoWrapper,
 } from "../../../common/Details";
-import { PosterWrapper, PosterDisplay, MainInfoWrapper, MainInfoTitle } from "../../movies/MovieDetails/styled";
+import Backdrop from "./Backdrop";
+import Loading from "../../../common/Loading";
+import Error from "../../../common/Error";
 import PeopleList from "../../people/PeopleList";
-import image from "../../../common/images/poster.png";
+import { changeMovieId, fetchMovieDetails, selectError, selectLoading, selectMovieDetails } from "./movieDetailsSlice";
 
 const MovieDetails = () => {
+  const dispatch = useDispatch();
+  const { id } = useParams();
+  const loading = useSelector(selectLoading);
+  const error = useSelector(selectError);
+  const movie = useSelector(selectMovieDetails);
+  const releaseDate = movie.release_date;
+  const [date, setDate] = useState("");
+
+  useEffect(() => {
+    dispatch(changeMovieId(id));
+    dispatch(fetchMovieDetails());
+  }, [id, dispatch]);
+
+  useEffect(() => {
+    if (releaseDate) {
+      const [year, month, day] = releaseDate.split("-");
+      setDate(day + "." + month + "." + year);
+    }
+  }, [releaseDate]);
+  
+  if (loading) {
```

Comment:
should be here loading and error? id someone click on movies there should be details as well

Reply:
I'm not sure if I understand it correctly, but I think that we need loading and error in most components.

Loading "just in case" and error is needed if there are problems with external api.